README.md

```diff
@@ -1,61 +1,53 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Portfolio - mobile menu
```

Comment:
# 💾 💾 💽 [OPTIONAL]   💽 💾 💾
**✔️ _Keep professionalism in your Repo's_** 
- Create to your ``assets`` a directory and keep clean your main(root) direcory.
- Kindly fill and update your ``README`` file.